include/spdk/nvme.h

```diff
@@ -90,6 +90,11 @@ struct spdk_nvme_ctrlr_opts {
 	 * are sent.
 	 */
 	uint32_t keep_alive_timeout_ms;
+	/**
+	 * Holds nvme timeout value. It is initialized to the value
+	 * of NVME_IO_TIMEOUT.
+	 */
+	uint32_t nvme_io_timeout;
```

Comment:
The units of this value should be documented (seconds? milliseconds?) and ideally represented in the name, like keep_alive_timeout_ms above.

Also, the default value doesn't need to be mentioned here (it is an implementation detail that may change, and the user can detect what the default is by looking at the ctrlr_opts when probe_cb is called).